### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by JAKAMI99** [henrywhitaker3/Speedtest-Tracker#491]
**Show time in the x graph*
It would be awesome if there where timestamps.

Because you can change the test-intervall via crontab it would be useful to make it zoomabl

**Comment by makitsune** [comment]
Yes please, this is really important. When I'm looking at the dl/up graph, I literally don't know what I'm looking at. It's equally unclear if I take a screenshot to send to someone.
